AirCasting/CoreData/PersistenceController+Database.swift

```diff
@@ -29,6 +29,11 @@ extension PersistenceController: SessionInsertable {
         let context = self.editContext
         context.perform {
             sessions.forEach {
+                    // This is added to ensure that we don't add session if a session with this uuid already existis in the database
+                    guard (try? context.existingSession(uuid: $0.uuid)) == nil else {
+                        Log.warning("Tried to add new session with same UUID")
```

Comment:
I think this should be an `error` log (the difference is that `error` logs are logged to firebase as non-crashing errors. I think we'd like to be informed of this case)